### ISSUE TITLE
Why does VIC5 require a minimum simulation period of 1 day?

### ISSUE THREAD

**Issue by Ahmed-Sheba** [UW-Hydro/VIC#951]
I have noticed that the model appears to have a restriction where the shortest simulation is 1 day. Is this limitation due to specific sub-models, such as the snow model, or does it affect multiple components of VIC5? If so, which components are involved?

I am currently working on a project that requires VIC5 simulations to run for shorter time periods, such as 1 hour. I would like to know if there are specific parts of the code that need to be adjusted to support this, or if it's related to limitations within certain model components (e.g., the snow model). Could you provide guidance on which files or settings would need modification to achieve this?

**Comment by guzaishang** [comment]
Have you solved this problem? I had the same problem.

**Comment by Ahmed-Sheba** [comment]
Yes. If you're interested in the details or need help implementing the fix, feel free to contact me at aas354@pitt.edu.
